src/editor/editor.ts

```diff
@@ -10,26 +10,42 @@ import {jsonAttr} from '@exadel/esl/modules/esl-base-element/core';
 
 import {UIPPlugin} from '../core/plugin';
 
+/** Config interface for customizing editor's behaviour. */
 interface EditorConfig {
+  /** Editor's appearance theme. */
   theme: string;
+  /**
+   * Text mode which is used
+   * inside the editor.
+   */
   mode: string;
+  /** Position of the vertical line for wrapping. */
   printMarginColumn: number;
+  /** Limit of characters before wrapping. */
   wrap: number;
 }
 
+/**
+ * Code editor for changing current markup. Allows user to manually configure
+ * the component inside {@link UIPPreview}.
+ * @see {@link UIPPlugin}
+ */
 export class UIPEditor extends UIPPlugin {
   public static is = 'uip-editor';
+  /** Default [config]{@link EditorConfig} used by editor. */
   public static defaultOptions = {
     theme: 'ace/theme/chrome',
     mode: 'ace/mode/html',
     printMarginColumn: -1,
     wrap: true,
   };
 
+  /** Editor's [config]{@link EditorConfig} passed through attribute. */
   @jsonAttr()
   public editorConfig: EditorConfig;
   protected editor: Ace.Editor;
 
+  /** Merge [default config]{@link defaultOptions} with editor's [config]{@link editorConfig}. */
   protected get mergedEditorConfig(): EditorConfig {
```

Comment:
Note: Regardless of the fact that the getter is a function, it defines the simple object property, so the description should build accordingly.
Also, fill free to use the special word `@returns` it helps to build description text for things that return something